docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -75,29 +81,32 @@ Key Term Examples  - Unobservable theoretical constructs and Measurement models
 
 - **Socioeconomic status** is an example of an unobservable theoretical construct. A common measurement model includes annual household income. 
 
-- **Patient benefit** is an example of an unobservable theoretical construct. A common measurement model involves patient care costs. See [#5]_ for a related example. 
+- **Patient benefit** is an example of an unobservable theoretical construct. A common measurement model involves patient care costs. See :footcite:`doi:10.1126/science.aax2342` for a related example. 
 
-**Note:** We cite several examples of unobservable theoretical constructs and measurement models for the purpose of explaining the key terms outlined above.  Please reference Measurement and Fairness [#4]_ for more detailed examples.
+**Note:**
+We cite several examples of unobservable theoretical constructs and measurement models for the purpose of explaining the key terms outlined above.
+Please reference :footcite:cts:`10.1145/3442188.3445901` for more detailed examples.
  
  
 .. _construct_validity:
 
 What is construct validity? 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Though the Measurement and Fairness [#4]_ paper explores both construct reliability and construct validity, we focus our 
-exploration below on construct Validity. We note that both play an important role in understanding fairness in sociotechnical 
-contexts. With that said, Measurement and Fairness [#4]_ offers a fairness-oriented conceptualization of construct validity, that 
-is helpful in thinking about fairness in sociotechnical contexts. We capture the idea in seven key parts that when combined 
-can serve as a framework for analyzing an AI task and attempting to establish construct validity:
+Though :footcite:cts:`10.1145/3442188.3445901` explore both construct reliability and construct validity, we focus our 
```

Comment:
Do we want the citation to use this complex ID rather than the simpler ines we've been using so far? I am more familiar with the simple ones like agarwal2016reductions.

Reply:
I prefer the format @romanlutz suggested! The format is easy to reproduce for other journals/conferences/ArXiv. I also expect that this format makes it less likely that people add redundant copies to the .bib.

Reply:
Can you clarify what you mean by 'format is easy to reproduce' @hildeweerts ?

Reply:
I prefer it for the simple reason that I can read the text without checking the literature section and I have a hunch which paper it is 🙂 Agarwal 2016 reductions tells me all I need to know. That number thingy... not so much.

I agree with "easy to reproduce" in terms of just taking first author's last name + year + first (real) word of title. I believe that's what @hildeweerts meant (?)

Reply:
I really think we should just copy/paste from our source and be done with it. I don't think citations warrant this much discussion 😁 I'm happy if we come up with a script that replaces all citations we have.

Reply:
> I prefer it for the simple reason that I can read the text without checking the literature section and I have a hunch which paper it is 🙂 Agarwal 2016 reductions tells me all I need to know. That number thingy... not so much.

Same!

> I agree with "easy to reproduce" in terms of just taking first author's last name + year + first (real) word of title. I believe that's what @hildeweerts meant (?)

Yeah, exactly :)

Reply:
I would agree that ACM is using a particularly obscure key (the ADS format at least gives you a chance of figuring out the paper). But as @adrinjalali says, in the long run, it will be easier just to copy/paste the citations from the source. And to catch duplicates, since BibTeX should whine if the same key appears twice (we might concoct our own 'approved' format, but if creation is manual then there's room for error).

Reply:
I don't think this will resolve duplicates, because some people may refer to the ArXiV preprint instead of a published paper and others may copy the citation from Google Scholar or something like doi2bib.org instead of the original publication venue.

[PS. I agree with @adrinjalali that citation probably do not warrant this much discussion but don't think it's that much effort to try to keep things consistent :D]